src/date/expand-pattern.js

```diff
@@ -33,10 +55,27 @@ return function( pattern, cldr ) {
 
 	switch ( true ) {
 		case "skeleton" in pattern:
+			skeleton = pattern.skeleton;
 			result = cldr.main([
 				"dates/calendars/gregorian/dateTimeFormats/availableFormats",
-				pattern.skeleton
+				skeleton
 			]);
+			if ( !result ) {
+				timeSkeleton = skeleton.split( /[^hHKkmsSAzZOvVXx]/ ).slice( -1 )[ 0 ];
+				dateSkeleton = skeleton.split( /[^GyYuUrQqMLlwWdDFgEec]/ )[ 0 ];
+				if (  /(MMMM|LLLL).*[Ec]/.test( dateSkeleton ) ) {
+					type = "full";
+				} else if ( /MMMM/g.test( dateSkeleton ) ) {
+					type = "long";
+				} else if ( /MMM/g.test( dateSkeleton ) || /LLL/g.test( dateSkeleton ) ) {
+					type = "medium";
+				} else {
+					type = "short";
+				}
+				result = getDateTime( type, dateSkeleton, timeSkeleton,
+						"dates/calendars/gregorian/dateTimeFormats/availableFormats",
+						"dates/calendars/gregorian/dateTimeFormats/availableFormats", cldr );
```

Comment:
Sir, I tried the code suggested by you and it worked great but there is a flaw in `combineDateTime()` function. I feel it should be `[ timePattern, datePattern ]` and not `[ datePattern, timePattern ]` comparing the results of previous code with new implementation. Hence committing this change as well. Let me know if its wrong.